src/pages/_app.tsx

```diff
@@ -23,13 +23,18 @@ export default function Intellect({ Component, pageProps }: AppPropsWithLayout)
     return (
         <>
             <Head>
-                <meta charSet="utf-8" />
+                <meta charSet="utf-8"/>
                 <title key="title">Інтелект | Викладачі та науковці</title>
 
-                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico' />
-                <link rel="preconnect" href="https://fonts.googleapis.com" />
-                <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin="anonymous" />
+                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico'/>
```

Comment:
Order of links matters.
Please copy the changes from schedule in the same order and with the same attributes as there.

Reply:
ordered icon links as it is done in schedule